docs/modules/Conch::Route::User.md

```diff
@@ -14,7 +14,7 @@ Unless otherwise noted, all routes require authentication.
 
 - Response: response.yaml#/UserDetailed
 
-### `POST /user/me/revoke?send_mail=<1|0>& login_only=<0|1> or ?api_only=<0|1>`
+### `POST /user/me/revoke?send_mail=<1|0>& login_only=<0|1> or &api_only=<0|1>`
```

Comment:
Probably should be `POST /user/me/revoke/?send_mail=<1|0>&login_only=<0|1>&api_only<0|1>` and not have the `or` in there. The text below covers the fact that they're mutually exclusive in a clearer way.